pros/conductor/conductor.py

```diff
@@ -279,13 +288,15 @@ def remove_template(project: Project, identifier: Union[str, BaseTemplate], remo
                                     remove_empty_directories=remove_empty_directories)
 
     def new_project(self, path: str, no_default_libs: bool = False, **kwargs) -> Project:
-        self.is_beta = kwargs.get('beta', False)
+        self.use_pros4 = kwargs.get('modern', False)
+        if not self.use_pros4 and self.warned_pros3_deprecated:
+            ui.echo(f"PROS 3 is deprecated. PROS 4 is now the latest version. "
```

Comment:
maybe here we can notify that PROS 4 is available as an option but PROS 3 is still the default